tests/src/providers/testqgspdalprovider.cpp

```diff
@@ -174,6 +189,27 @@ void TestQgsPdalProvider::validLayer()
 
   QCOMPARE( layer->dataProvider()->pointCount(), 253 );
   QCOMPARE( layer->pointCount(), 253 );
+  QVERIFY( layer->dataProvider()->indexingState() == QgsPointCloudDataProvider::NotIndexed );
+}
+
+void TestQgsPdalProvider::testEptGeneration()
+{
+  QTemporaryDir dir;
+  QVERIFY( dir.isValid() );
+  QgsPdalEptGenerationTask task( mTestDataDir + QStringLiteral( "point_clouds/las/cloud.las" ), dir.path() );
+  QVERIFY( task.run() );
+  QFileInfo fi( dir.path() + "/ept.json" );
+  QVERIFY( fi.exists() );
+}
+
+void TestQgsPdalProvider::testEptGenerationNonASCII()
+{
+  QTemporaryDir dir;
+  QVERIFY( dir.isValid() );
+  QgsPdalEptGenerationTask task( mTestDataDir + QStringLiteral( "point_clouds/las/cloud%!* _*čopy.las" ), dir.path() );
```

Comment:
Why was this a good filename? This literally breaks git on windows and prevent changing branch. Renaming this would solve the issue.